graphql/executor.go

```diff
@@ -158,7 +170,17 @@ func (e *Executor) resolveAndExecute(ctx context.Context, field *Field, source i
 			}
 
 			// TODO: Consider cacheing resolve and execute independently
+			fromCache := true
```

Comment:
This `fromCache` thing is clever, but it's a little opaque. Maybe we could add an overload to reactive.Cache that returns if it was cached or not?